projects/core/src/lib/project/project.service.ts

```diff
@@ -850,6 +850,20 @@ export class ProjectService {
         return this.selectedFeature$.value;
     }
 
+    /**
+     * Get a stream that signals whether a running query should be aborted because the results are no longer needed
+     */
+    getQueryAbortStream(): Observable<any> {
```

Comment:
"create" since you can only use it once?

Reply:
cdcf2d8422c2a68248a0fd29e85970fad57ad68d